src/requests/login.js

```diff
@@ -5,5 +5,5 @@ const server = require('../../server');
 
 module.exports = function (data) {
   if (auth.sessionValid(data.authUserID, data.authSession)) data.res.redirect("/");
-  else data.res.redirect("https://discordapp.com/api/oauth2/authorize?client_id=591694201230721043&redirect_uri=https%3A%2F%2Fdlockly.glitch.me%2Fauth&response_type=code&scope=identify");
+  else data.res.redirect("https://discordapp.com/api/oauth2/authorize?client_id="+server.bot.user.id+"&redirect_uri=https%3A%2F%2F"+process.env.PROJECT_DOMAIN+".glitch.me%2Fauth&response_type=code&scope=identify");
```

Comment:
Also the same as other comment but made the id work too. This means that the login page won't work if the bot is not signed in